test/dynamo/test_dynamic_shapes.py

```diff
@@ -29,8 +29,6 @@
     "ReproTests": [
         # Could not infer dtype of torch._C.SymIntNode
         "test_convert_boxes_to_pooler_format",
-        # Cannot call sizes() on tensor with symbolic sizes/strides
-        "test_hf_t5_forward",
```

Comment:
The comment above is misleading. This was failing for a while due to the difference in `op_count` (call count).

The only difference is the call to `_dynamo_get_item_lambda`, which doesn't seem to be used.

Before (when this was passing) vs now (before this PR):
```
 def forward(self, hidden_states : torch.Tensor):
     size = hidden_states.size()
-    getitem = size[0];  size = None
+    getitem = size[0]
+    _dynamo_get_item_lambda = torch__dynamo_variables_lists__dynamo_get_item_lambda(size, slice(None, 2, None));  size = None
     self_q = self.self_q(hidden_states)
     view = self_q.view(getitem, -1, 8, 64);  self_q = None
     transpose = view.transpose(1, 2);  view = None
     self_k = self.self_k(hidden_states)
     view_1 = self_k.view(getitem, -1, 8, 64);  self_k = None
     transpose_1 = view_1.transpose(1, 2);  view_1 = None
     self_v = self.self_v(hidden_states);  hidden_states = None
     view_2 = self_v.view(getitem, -1, 8, 64);  self_v = None
     transpose_2 = view_2.transpose(1, 2);  view_2 = None
     transpose_3 = transpose_1.transpose(3, 2);  transpose_1 = None
     matmul = torch.matmul(transpose, transpose_3);  transpose = transpose_3 = None
     return (matmul, transpose_2, getitem)

 # To see more debug info, please use `graph_module.print_readable()`
 call_method size
 call_function <built-in function getitem>
+call_function <function SizeVariable.get_item_dyn.<locals>._dynamo_get_item_lambda at 0x7f63e0b781f0>
 call_module self_q
 call_method view
 call_method transpose
 call_module self_k
 call_method view
 call_method transpose
 call_module self_v
 call_method view
 call_method transpose
 call_method transpose
-call_function <built-in method matmul of type object at 0x7fab4f0ad0c0>
+call_function <built-in method matmul of type object at 0x7f64b7ad5860>
```